scale/scale/settings.py

```diff
@@ -29,6 +29,9 @@
 # The full name for the Scale Docker image (without version tag)
 SCALE_DOCKER_IMAGE = 'geoint/scale'
 
+# The location of the config file containing Docker credentials
+CONFIG_URI = os.environ.get('CONFIG_URI', None)
```

Comment:
You also need to change the dockerfile and entrypoint script so this can be passed when executing the docker container. In addition, you'll probably need to update the DC/OS deployment configuration to include a DC/OS option to set this value. issue-15 (when it's merged) checks in a copy of the DC/OS config where you can make the change.